manifests/params.pp

```diff
@@ -53,6 +53,8 @@
         'LogSyslog'      => 'yes',
         'DatabaseMirror' => 'database.clamav.net',
       }
+      $freshclam_sysconfig = '/etc/sysconfig/freshclam'
+      $freshclam_delay     = 'disabled-warn'
```

Comment:
```disabled-warn``` is what the rpm puts in here by default right?
This effectively disables updates...  What do you change this to to get it working?
Should we maybe put something better here, so that it works out of the box.

I think the general consensus is that the defaults for RHEL 7 are pretty bad and should be changed to something that produces a working system out of the box...
Were working on the config file defaults as well to this end.